docs/pages/access-controls/access-request-plugins/servicenow.mdx

```diff
@@ -35,6 +37,35 @@ API.
 Before making the access request ensure the user making the request has
 the `requester` role.
 
+For the plugin to know which ServiceNow rotations to check for the
+auto approval flow, the rotation you wish users to be checked against
+must be included in one of the requesting user's roles. To do so, add the `teleport.dev/schedules` annotation as shown in the following example: 
+
+```yaml
+kind: role
+version: v5
+metadata:
+  name: YOUR_ROLE_NAME_HERE
+spec:
+  allow:
+    request:
+      roles: ['editor']
+      thresholds:
+        - approve: 1
+          deny: 1
+      annotations:
+        teleport.dev/schedules:
+        - YOUR_SERVICENOW_ROTA_ID_HERE
+```
+
+To retrieve the ServiceNow rotation ID, first navigate to the group record
+ of the ServiceNow group the rotation belongs to and right click on
+ header->'Select copy sys_id' to copy the sys_id.
+
+Then using the ServiceNow endpoint '/api/now/on_call_rota/workbench/group/{groupSysId}'
```

Comment:
This still feels hard to follow. How am I supposed to hit the endpoint? Do I need credentials?

Reply:
Good point. Would an example curl request be better?